sale_pricelist_triple_discount/model/sale.py

```diff
@@ -1,55 +1,192 @@
 # Copyright 2019 Simone Rubino - Agile Business Group
+# Copyright (c) 2021 Andrea Cometa - Apulia Software s.r.l.
+# Copyright 2022 Alberto Re - Agile Business Group
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from openerp import api, models
+from functools import partial
+
+from odoo import api, models, tools
+from odoo.tools.misc import formatLang
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    def _amount_by_group(self):
```

Comment:
Could you please clarify why is this needed? I don't see the new fields **discount*** used in this override